lib/loadObj.js

```diff
@@ -135,6 +152,94 @@ function loadObj(objPath, options) {
         primitive.material = getName(name);
     }
 
+    var intPoint = new Cartesian3();
+    var xAxis = Cesium.Cartesian3.UNIT_X.clone();
+    var yAxis = Cesium.Cartesian3.UNIT_Y.clone();
+    var zAxis = Cesium.Cartesian3.UNIT_Z.clone();
```

Comment:
Do these need be cloned from `UNIT_X` and others? I think they can just be `new Cesium.Cartesian3()`.